eventuals/grpc/client.h

```diff
@@ -305,6 +305,12 @@ class Client {
     : channel_(::grpc::CreateChannel(target, credentials)),
       pool_(std::move(pool)) {}
 
+  Client(
```

Comment:
Generally, all constructors should be `explicit` (probably the constructor above, too): https://abseil.io/tips/142

Reply:
That's a great advice! We should also add `explicit` in other parts of eventuals as well.

Reply:
👍🏻